docs/data.md

```diff
@@ -1,40 +1,60 @@
+As such, `scores` works with any data source for which xarray or pandas can be used.
 
-## Gridded global numerical weather prediction data
-Global weather prediction models are used to generate medium range forecasts and provide the initial and boundary conditions for higher-resolution regional models. Their global coverage makes them a good starting point for demonstrating the application of scoring methods in any region of interest.
+Users will need to supply the dataset(s) they wish to work with, as `scores` does not contain datasets.
 
-The Bureau of Meteorology provides global model data from the ACCESS numerical weather prediction system. See https://geonetwork.nci.org.au/geonetwork/srv/eng/catalog.search#/metadata/f3307_5503_1483_3079 for more information.
+Data referred to on this page is available under various licenses, and the onus is on the user to understand the conditions of those licenses.
 
-Global model data is also available from the NOAA Global Forecast System (GFS). See https://www.ncei.noaa.gov/products/weather-climate-models/global-forecast for more information.
+For additional information about downloading and preparing sample data, see [this tutorial](project:./tutorials/First_Data_Fetching.md).
 
-## CliMetLab
-ECMWF has developed the CliMetLab python package to simplify access to a large range of climate and meteorological datasets. See https://climetlab.readthedocs.io/en/latest/
+## Working with Different File Formats
 
-## NOAA ISD dataset
-The NOAA Integrated Surface Database provides hourly point-based (aka in-situ) data globally and is a good starting point for understanding how to work with point-based data. Point-based observations are shared routinely between countries for the purposes of weather modelling.
+### Working with GRIB Data
 
-See https://www.ncei.noaa.gov/products/land-based-station/integrated-surface-database for more information.
+To use `scores` with [GRIB](https://codes.wmo.int/grib2) data, install [cfgrib](https://github.com/ecmwf/cfgrib) and use `engine='cfgrib'` when opening a GRIB file with xarray.
 
-## Gridded model reanalysis data
-Reanalysis data is useful for providing a very long history of atmospheric conditions. The ERA5 dataset is a well known and widely used global reanalysis dataset.
+### Working with NetCDF Data
 
-For more information see https://www.ecmwf.int/en/forecasts/datasets/reanalysis-datasets/era5 and
-https://github.com/pangeo-data/WeatherBench
+To use `scores` with [NetCDF](https://doi.org/10.5065/D6H70CW6) or [HDF5](https://doi.org/10.11578/dc.20180330.1) data, install [h5netcdf](https://github.com/h5netcdf/h5netcdf). The h5netcdf library is included in the `scores` ["all"](installation.md#all-dependencies-excludes-some-maintainer-only-packages) and ["tutorial"](installation.md#tutorial-dependencies) installation options. Opening NetCDF data is demonstrated in [this tutorial](project:./tutorials/First_Data_Fetching.md). 
```

Comment:
On further reflection, the HDF5 link I included was the one used in the JOSS draft (and is probably fine for that purpose). However, it points to a specific release, which doesn't appear to be the latest release. Rather than needing to stay abreast of the HDF5 release cycle, my guess is for this specific context, it would be better to replace the HDF5 link with something else - perhaps their GitHub repo?

Reply:
I think on balance the github link is the most likely to be the most useful link for users of this page

Reply:
I have updated to the GitHub link